src/OpenTelemetry/Trace/Export/Internal/MultiProcessor.cs

```diff
@@ -0,0 +1,60 @@
+﻿// <copyright file="MultiProcessor.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenTelemetry.Trace.Export
+{
+    internal class MultiProcessor : SpanProcessor
+    {
+        private readonly IEnumerable<SpanProcessor> processors;
+
+        public MultiProcessor(IEnumerable<SpanProcessor> processors)
+        {
+            this.processors = processors ?? throw new ArgumentNullException(nameof(processors));
+        }
+
+        public override void OnEnd(Span span)
+        {
+            foreach (var processor in this.processors)
```

Comment:
so the order of Start and End the same? Should it be opposite?

Reply:
spec says they should be executed in the same order they've been added.

Don't forget we have multiple processors on Tracer. And each processor may be chained. 
Multi-processor is beginning of each chain

Reply:
ok. Found it in specs. Since there is no shared state - doesn't really matter the ordering between Start and End